apps/remix-ide/src/app/tabs/compileTab/compilerContainer.js

```diff
@@ -476,6 +476,15 @@ class CompilerContainer {
         this._view.versionSelector.appendChild(option)
       }
     })
+    if (semver.lt(this._retrieveVersion(), 'v0.4.12+commit.194ff033.js')) {
+      toaster(yo`
+        <div>
+          <b>Old compiler usage detected.</b>
+          <p>You are using a compiler older than v0.4.12.</p>
+          <p>Some Functionallity may not work.</p>
```

Comment:
Some functionality may not work.

(you have an extra l in functionality)  Also the f should be lowercase.

Reply:
thanks